pkg/system/stat_solaris.go

```diff
@@ -0,0 +1,13 @@
+package system // import "github.com/docker/docker/pkg/system"
+
+import "syscall"
+
+// fromStatT converts a syscall.Stat_t type to a system.Stat_t type
+func fromStatT(s *syscall.Stat_t) (*StatT, error) {
+	return &StatT{size: s.Size,
```

Comment:
Please gofmt

Reply:
Huh, I just ran gofmt and that did not change a thing. Also it looks gofmt to my eye. If you talk about the `&StatT{size` part that is technically valid and accepted. And the same in all other files inside this package. Which IMHO are out of scope for this PR. But I could reformat that in another if you'd like.

Reply:
Ah, didn't know this is valid..

Reply:
Yeah I don't consider it valid personally but gofmt does. I change it in my own projects but here all other files have it like this so I prioritized consistency over personal preference.